src/components/textbox/textbox.component.tsx

```diff
@@ -233,7 +233,7 @@ export const Textbox = React.forwardRef(
       error,
       warning,
       info,
-      label,
+      label: "fill in all fields marked with *",
```

Comment:
Question: why are we not passing the label text passed by the implementing teams?